WooCommerce/WooCommerceTests/ViewRelated/Orders/Issue Refund/RefundConfirmationViewModelTests.swift

```diff
@@ -43,12 +43,48 @@ final class RefundConfirmationViewModelTests: XCTestCase {
                                                 numberOfDecimals: 2)
 
         let order = MockOrders().empty()
-        let details = RefundConfirmationViewModel.Details(order: order, amount: "130.3473", refundsShipping: false, items: [])
+        let details = RefundConfirmationViewModel.Details(order: order, amount: "130.3473", refundsShipping: false, items: [], paymentGateway: nil)
 
         // When
         let viewModel = RefundConfirmationViewModel(details: details, currencySettings: currencySettings)
 
         // Then
         XCTAssertEqual(viewModel.refundAmount, "$130.35")
     }
+
+    func test_viewModel_has_automatic_refundVia_values_when_using_a_gateway_that_support_refunds() throws {
+        // Given
+        let order = MockOrders().empty().copy(paymentMethodID: "stipe", paymentMethodTitle: "Stripe")
+        let gateway = PaymentGateway(siteID: 123, gatewayID: "stripe", title: "Stripe", description: "", enabled: true, features: [.refunds])
+        let details = RefundConfirmationViewModel.Details(order: order, amount: "", refundsShipping: false, items: [], paymentGateway: gateway)
+
+        // When
+        let viewModel = RefundConfirmationViewModel(details: details)
+
+        // We expect the Refund Via row to be the last item in the last row.
+        let row = try XCTUnwrap(viewModel.sections.last?.rows.last as? RefundConfirmationViewModel.TitleAndBodyRow)
+
+        // Then
+        XCTAssertEqual(row.title, gateway.title)
```

Comment:
nit: maybe comparing against `order.paymentMethodTitle` would be the most accurate? (like in `RefundConfirmationViewModel.makeRefundViaRow`)

```suggestion
        XCTAssertEqual(row.title, order.paymentMethodTitle)
```